src/api/app/assets/stylesheets/webui/application/dialog.scss

```diff
@@ -40,6 +40,17 @@
       font-weight: bold;
       font-size: 1.2em;
     }
+    .close_dialog {
```

Comment:
`#remove_dialog` is an id and since we have multiple dialogs on that page, we cannot use it. We also cannot use it cause the "#remove_dialog" is used for removing the dialogs completely from the DOM. This is why I introduced the hiding/showing functionality. Especially in this case, it makes sense.